DevelopGitHub/GitHubExtension/GitHubExtension.Statistics.WebApi/Mappers/HttpRequest/CreateMessageMapper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Net.Http;
+using GitHubExtension.Statistics.WebApi.Mappers.Constant;
+
+namespace GitHubExtension.Statistics.WebApi.Mappers.HttpRequest
+{
+    public static class CreateMessageMapper
```

Comment:
It's not a mapper and it's not extend HttpMethod. It should accept HttpRequestMessage and named as HttpRequestMessageExtension.